### PR TITLE
added german translation

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
refs #88

File: app/Resources/translations/messages.de.xliff [hunk 1]
```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
```
Comment (javiereguiluz) [inline app/Resources/translations/messages.de.xliff]: `target-language="en"` -> `target-language="de"` (I always forget to update this value too)

**Comment by OskarStark** [comment]
done, thank you :smile:

File: app/Resources/translations/messages.de.xliff [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+                <source>tip</source>
+                <target>TIPP</target>
+            </trans-unit>
+            <trans-unit id="not_available">
+                <source>not_available</source>
+                <target>Nicht verfügbar</target>
+            </trans-unit>
+            <trans-unit id="mit_license">
+                <source>mit_license</source>
+                <target>MIT Lizenz</target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xliff]: MIT-Lizenz

File: app/Resources/translations/messages.de.xliff [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+            </trans-unit>
+
+            <trans-unit id="title.homepage">
+                <source>title.homepage</source>
+                <target><![CDATA[Willkommen zur <strong>Symfony Demo Applikation</strong>]]></target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xliff]: Maybe this should be Application if we don't translate the application name.

File: app/Resources/translations/messages.de.xliff (hunk 2 above)
Comment (OskarStark) [inline app/Resources/translations/messages.de.xliff]: done

File: app/Resources/translations/messages.de.xliff (hunk 3 above)
Comment (OskarStark) [inline app/Resources/translations/messages.de.xliff]: hmm i dont think so, what is done in the other translation files, didnt have look there

@javiereguiluz what do you think

Comment (javiereguiluz) [inline app/Resources/translations/messages.de.xliff]: I don't have a clear opinion about this. See https://github.com/symfony/symfony-demo/pull/85 for further discussion.

Comment (OskarStark) [inline app/Resources/translations/messages.de.xliff]: ok i changed it to the english default.

later on we can change it if we like

**Comment by javiereguiluz** [comment]
@OskarStark awesome! Thanks for this new translation for the demo application. *danke schön*

**Comment by OskarStark** [comment]
:)

**Pull request closed without merging**